test/spectral/tests/custom-rules/servers-variable-default-value.test.js

```diff
@@ -69,9 +69,66 @@ describe('spectral - test whether servers variables have default value', () => {
       paths: {}
     };
     const res = await inCodeValidator(spec, true);
-    const result = res.warnings.find(
-      elem => elem.rule === 'server-variable-default-value'
+    const errorMessageFind = res.warnings.find(
+      elem => elem.message === 'Server variable should have default value'
+    );
+    expect(errorMessageFind).not.toBeUndefined();
+
+    const expectedPath = ['servers', '0', 'variables', 'name', 'default'];
+    expect(errorMessageFind.path.sort().join('')).toBe(
+      expectedPath.sort().join('')
     );
-    expect(result).not.toBeUndefined();
+  });
+
+  it('should show warning when multiple server variable default value does not have default value', async () => {
+    const spec = {
+      openapi: '3.0.0',
+      info: {
+        version: '1.0.0',
+        title: 'Server variable does not have default value'
+      },
+      servers: [
+        {
+          url: 'https://example.com',
+          description:
+            'Since the variable does not have value a warning will be displayed',
+          variables: {
+            name: {
+              default: ''
+            },
+            address: {
+              default: ''
+            }
+          }
+        }
+      ],
+      paths: {}
+    };
+    const res = await inCodeValidator(spec, true);
+    const errorMessageFilter = res.warnings.filter(
+      elem => elem.message === 'Server variable should have default value'
+    );
+    expect(errorMessageFilter.length).toBe(2);
+
+    let pathHits = 0;
```

Comment:
Sorry to be picky here! I think it's simpler to utilize the jest tooling here:

```suggestion
    expect(res.warnings[0].path.toEqual(['servers', '0', 'variables', 'name', 'default'])
    expect(res.warnings[1].path.toEqual(['servers', '1', 'variables', 'address', 'default'])
```

Reply:
Ahh, awesome!